src/vpoller/vsphere/tasks.py

```diff
@@ -1987,7 +1988,16 @@ def vm_guest_net_get(agent, msg):
     # Get the requested properties
     result = {}
     result['name'] = vm_name
-    result['net'] = [{prop: getattr(net, prop, '(null)') for prop in properties} for net in vm_networks]
+        result['net'] = []
```

Comment:
Yes, I make a mistake.
Please ignore this pull request, I do not know well "Github" and I made a "pull request" by mistake